packages/components/src/Headline/Headline.tsx

```diff
@@ -9,31 +10,23 @@ import {
   useClassNames,
   useTheme,
 } from '@marigold/system';
-import { HtmlProps } from '@marigold/types';
 
-// Props
-// ---------------
+type RemovedProps = 'className';
 export interface HeadlineProps
-  extends Omit<HtmlProps<'h1'>, 'className'>,
+  extends Omit<RAC.HeadingProps, RemovedProps>,
     TextAlignProp {
-  children?: ReactNode;
-  level?: '1' | '2' | '3' | '4' | '5' | '6';
   color?: string;
   variant?: string;
   size?: string;
-  as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
 }
 
-// Component
-// ---------------
-export const Headline = ({
-  children,
+const _Headline = ({
   variant,
   size,
+  children,
   align = 'left',
   color,
-  level = '1',
```

Comment:
Hm. should we allow both so this is not a breaking change?

Reply:
But than I can't use the prop from RAC oder?👀 @sebald

Reply:
You can cast it with `Number(level)`. If we only allow 1-6 as string plug 1-6 as number this should be safe :)

Reply:
done please have a loook :)